cli/api/commands/prune.ts

```diff
@@ -44,6 +44,7 @@ function computeIncludedActionNames(
 
   const hasActionSelector = runConfig.actions?.length > 0;
   const hasTagSelector = runConfig.tags?.length > 0;
+  const hasFlterTagsAsIntersectionSelector = runConfig.filterTagsAsIntersection?.valueOf();
```

Comment:
Instead of `runConfig.filterTagsAsIntersection?.valueOf()` I suggest to use `!!runConfig.filterTagsAsIntersection`

There is also a typo in the word `Filter`.